omics/uriparse/uri_parse.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+from urllib.parse import urlparse
+
+from omics.common.omics_file_types import OmicsUriType, omicsUriType_to_FileNames
+
+
+class OmicsUri:
+    """Class for representing an omics URIs.
```

Comment:
Grammar: either: `Class for representing an Omics URI.`, or: `Class for representing Omics URIs.`